yojson-biniou.opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
```

Comment:
Shouldn't we get rid of that file?

Reply:
And all yojson-biniou related files actually

Reply:
👍